### PR TITLE
Select patient with command line arguments (limit user interaction)

### PR DESCRIPTION
**Pull request by cvermot**
Give patient ID as argument.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,22 @@ Run:
...
+```
+$ ./doctoshotgun.py paris roger.philibert@gmail.com PASSWORD
+Available patients are:
+* [0] Roger PHILIBERT
+* [1] Luce PHILIBERT
+You want to book a slot for whom patient?
+```
+You can also give the patient id as argument :
+```
+$ ./doctoshotgun.py paris roger.philibert@gmail.com PASSWORD - p 1
```
Comment (nicogodet) [inline README.md]:
```suggestion
$ ./doctoshotgun.py paris roger.philibert@gmail.com PASSWORD -p 1
```

**Comment by cvermot** [comment]
Thank you !

File: doctoshotgun.py [hunk 2]
```diff
@@ -370,25 +378,34 @@ def main(self):
             return 1
 
         patients = docto.get_patients()
+        #log(args.patient)
```
Comment (rbignon) [inline doctoshotgun.py]: You can remove this line

File: doctoshotgun.py [hunk 3]
```diff
@@ -206,6 +207,12 @@ def get_patients(self):
         self.master_patient.go()
 
         return self.page.get_patients()
+    
+    def normalize(self, string):
+        nfkd = unicodedata.normalize('NFKD', string)
+        normalized = u"".join([c for c in nfkd if not unicodedata.combining(c)])
+        normalized = re.sub(r'\W', '-', normalized)
+        return normalized.lower()
```
Comment (rbignon) [inline doctoshotgun.py]: Not related to this PR

File: doctoshotgun.py [hunk 4]
```diff
@@ -370,25 +378,34 @@ def main(self):
...
             docto.patient = patients[0]
 
-        cities = args.city.lower().split(',')
+        cities = (args.city).split(',')
+        for i, city in enumerate(cities):
+            cities[i] = docto.normalize(city)
         while True:
             for center in docto.find_centers(cities):
-                if center['city'].lower() not in cities:
+                if docto.normalize(center['city']) not in cities:
```
Comment (rbignon) [inline doctoshotgun.py]: Not related to this PR

File: README.md [hunk 5]
```diff
@@ -46,6 +46,22 @@ Run:
...
 ```
 
+For doctolib accounts with more thant one patient, you can select patient just after launching the script :
+
+```
+$ ./doctoshotgun.py paris roger.philibert@gmail.com PASSWORD
+Available patients are:
+* [0] Roger PHILIBERT
+* [1] Luce PHILIBERT
+You want to book a slot for whom patient?
```
Comment (rbignon) [inline README.md]: The wording has changed since last commit :)

**Review by rbignon — CHANGES_REQUESTED** [review]
Hello,

Thank you for your contribution. Can you rebase on master and remove commits not related to this PR?

**Comment by cvermot** [comment]
Should be fine now ! Thank you for reviewing.

**Pull request merged**